ports/spimpl/unofficial-spimpl-config.cmake

```
@@ -0,0 +1,7 @@
+add_library(unofficial::spimpl::spimpl INTERFACE IMPORTED)
+
+set_target_properties(
+  unofficial::spimpl::spimpl
+  PROPERTIES
+    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/../../include"
+)
```

The config might be loaded multiple times, so we need a guard:
```suggestion
if(NOT TARGET unofficial::spimpl::spimpl)
  add_library(unofficial::spimpl::spimpl INTERFACE IMPORTED)
  set_target_properties(
    unofficial::spimpl::spimpl
    PROPERTIES
      INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/../../include"
  )
endif()
```